fluidsynth/src/sfloader/fluid_defsfont.h

```diff
@@ -435,7 +435,7 @@ struct _fluid_defpreset_t
 {
   fluid_defpreset_t* next;
   fluid_defsfont_t* sfont;                  /* the soundfont this preset belongs to */
-  char name[21];                        /* the name of the preset */
+  char name[22];                        /* the name of the preset */
```

Comment:
Well, of course we can internally maintain names longer than what we actually read from the Soundfont file (as long as we only read and never write SF2). But do you really want such a special case **only** for the purpose of assigning a default name for that preset? Any other preset names will always fit in 20+1 bytes. I think this default preset name assignment is such a minor special case that it would be nice if it would simply stand to the rules.
What about shortening the default name to `Bank%d,Pre%d` or simply assign or leave it an empty string? I mean, if the incoming SF2 preset name is empty, why tinker with it and not simply leave it empty?